mbbsd/pfterm.c

```diff
@@ -2188,9 +2200,7 @@ fterm_rawscroll (int dy)
     // so don't use fterm_move*.
     if (dy > 0)
     {
-        // use a large row number in case terminal height > ft.rows
-        // The terminal height passed to resizeterm() can be unreliable
-        fterm_rawcmd2(9999, 1, 1, 'H');
+        fterm_rawcmd2(ft.rows_win, 1, 1, 'H');
```

Comment:
The API is designed to follow curses / ncurses. In that context, "terminal" means just that, the whole terminal. It does not and should not know about anything outside of the terminal, whether it be your GUI window menus and decoration, or an actual monitor. "Window" in this context means any subdivision of the terminal.

I guess you could keep using `ft.rows` and `ft.cols` internally to mean the "window" size to avoid changing all the references, and add something like `ft.rows_full` for the non-cropped size.